workflows/hifi2genome.nf

```diff
@@ -46,6 +40,23 @@ for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true
 */
 
 workflow HIFI2GENOME {
+    ch_versions = Channel.empty()
+
+    // read in samplesheet
+    ch_reads = INPUT_CHECK(ch_input)
+    ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
+
+    // assembly with flye
+    FLYE (
+        ch_reads.reads, //had to explicitly define .reads here even though module says it wants tuple(meta), path(reads)
```

Comment:
This comment is a lil redundant because `ch_reads.reads` is actually of type `tuple(meta, path(reads))`. I'd rename `ch_reads` on line 46 to `ch_data` or smth like that. And do `ch_data.reads` here. Alternatively, you can scrap `ch_reads`/`ch_data` fully and just do: `FLYE(INPUT_CHECK(ch_input).out.reads,...)` or smth like that here.